contrib/sssd.spec.in

```diff
@@ -37,7 +37,7 @@
 %global build_kcm_renewals 0
 %endif
 
-%if 0%{?fedora} >= 39
+%if 0%{?fedora} >= 39 || 0%{?rhel} >= 10
```

Comment:
Imo, make sense to use ">= 9" for sake of consistency, even if this file won't be actually used by RHEL9.

Reply:
Done